sys/net/routing/nhdp/iib_table.h

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright (C) 2014 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     nhdp
+ * @{
+ *
+ * @file
+ * @brief       Interface Information Base interface for NHDP
+ *
+ * @author      Fabian Nack <nack@inf.fu-berlin.de>
+ */
+
+#ifndef IIB_TABLE_H_
+#define IIB_TABLE_H_
+
+#include "timex.h"
+#include "kernel_types.h"
+
+#include "nib_table.h"
+#include "nhdp_address.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   Possible L_STATUS values of a link tuple
+ */
+typedef enum iib_link_tuple_status_t {
+    IIB_LT_STATUS_PENDING,
+    IIB_LT_STATUS_LOST,
+    IIB_LT_STATUS_HEARD,
+    IIB_LT_STATUS_SYM,
+    IIB_LT_STATUS_UNKNOWN
+} iib_link_tuple_status_t;
+
+/**
+ * @brief   Link Set entry (link tuple)
+ */
+typedef struct iib_link_set_entry_t {
+    nhdp_addr_entry_t* address_list_head;       /**< Pointer to head of this lt's addresses */
```

Comment:
Coding conventions: nhdp_addr_entry_t *address_list_head; 
Further comments on this style issue are suppressed for the rest of the review.